src/main/java/org/mockbukkit/mockbukkit/world/WorldMock.java

```diff
@@ -3450,8 +3452,19 @@ public void setSendViewDistance(int viewDistance)
 	@Override
 	public long getGameTime()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return this.gameTime;
+	}
+
+	/**
+	 * Sets the full in-game time on this world since the world generation.
+	 *
+	 * @param time The current absolute time since the world generation.
+	 */
+	@ApiStatus.Internal
```

Comment:
Where is this being used internally?

I usually argue that we should expose as litle as possible, but in this case, is it a good idea to expose this to the api?

Reply:
At the moment it's not, but i'm working on the villager entity i'll need that 😁🤣

Reply:
Ah. Then this is alright